docs/INSTALLATION.md

```diff
@@ -1,2 +1,13 @@
 # Installation
 
+Install the essential build tools, the QEMU emulator, and the GCC ARM64 cross-toolchain to test your code.
+
+```shell
+$ sudo apt install build-essential qemu-user qemu-user-static binutils-aarch64-linux-gnu gcc-aarch64-linux-gnu
+```
+
+The makefile should be able to handle the rest.
```

Comment:
Which makefile is this referring to?

Reply:
Probably the one already in the exercise directories. Should be made specific :)